### PR TITLE
bluetooth_battery: init at 1.2.0

### PR DESCRIPTION
**Pull request by cheriimoya**
###### Motivation for this change

I have a bluetooth headset that's capable of reporting the battery state. This tool allows for reading the battery percentage from my bluetooth headphones. I'd like to have this upstream, because i use it in my statusbar and it may be useful to many people.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by cheriimoya** [comment]
cc @ajs124

File: pkgs/applications/misc/bluetooth_battery/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{ lib, fetchFromGitHub, buildPythonPackage, pybluez }:
+
+buildPythonPackage rec {
```
Comment (mweinelt) [inline pkgs/applications/misc/bluetooth_battery/default.nix:3]: This does not look like a library to me, but rather a case for `buildPythonApplication`.

Comment (cheriimoya) [reply pkgs/applications/misc/bluetooth_battery/default.nix]: true, must have missed that when copying the expression from another package

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 103023` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>bluetooth_battery</li>
  </ul>
</details>

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -1483,6 +1483,12 @@
     githubId = 33503784;
     name = "Yucheng Zhang";
   };
+  cheriimoya = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:1486]: Please put this into a separate commit with the message ``maintainers: add cheriimoya``.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: maintainers/maintainer-list.nix (hunk 2 above)
Comment (cheriimoya) [reply maintainers/maintainer-list.nix:1586]: missed that one

**Review by mweinelt — APPROVED** [review]
The software seems a bit rough around the edges, but generally seems to work.

```
$ ./results/bluetooth_battery/bin/bluetooth_battery 
Usage: bl_battery.py <BT_MAC_ADDRESS_1>[.PORT] ...
         Port number is optional
```

Easy enough. But what do they mean by a port? And where to I find the MAC of my bluetooth devices?

```
$ ./results/bluetooth_battery/bin/bluetooth_battery  --help
<blocks>
```

Fine, `bluetoothctl` has those. My headphones are connected and I'm listening to music. Let's see how this goes:

```
$ ./results/bluetooth_battery/bin/bluetooth_battery CC:98:8B:XX:XX:XX
CC:98:8B:XX:XX:XX is offline (111, 'Connection refused')
$ ./results/bluetooth_battery/bin/bluetooth_battery CC:98:8B:XX:XX:XX
Battery level for CC:98:8B:XX:XX:XX is 50%
```

So yeah, seems to work generally. I'd love me some upower integration here. Until then, let's use this. Thanks for packaging it up, @cheriimoya.

**Comment by cheriimoya** [comment]
mmh yea, so the maintainer of the original code responded to [a recent issue of mine](https://github.com/TheWeirdDev/Bluetooth_Headset_Battery_Level/issues/43#issuecomment-752930586), i think you have the same issue right there...

[this fork](https://github.com/TheWeirdDev/Bluetooth_Headset_Battery_Level/compare/master...barak:master?diff=unified) has a nice readme update on how to find the bluetooth MACs

i didn't quite get the "port" part of the project either, i think it's more like a bluetooth channel (i guess?)

**Comment by mweinelt** [comment]
> mmh yea, so the maintainer of the original code responded to a recent issue of mine, i think you have the same issue right there...

also the same headphones :smile: 

Thanks!
